csv-exporter/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "csv-exporter",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "Export CKAN datasets to csv",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

If you're going to go to the bother of having an `.eslintrc` then you might as well use it.
```suggestion
    "test": "eslint ."
```

---

I mean, I am using it by dint of it being there, but I take your point!